includes/class-convertkit-resource-products.php

```diff
@@ -74,6 +74,16 @@ public function get_commerce_js_url() {
 		// Parse the URL.
 		$parsed_url = wp_parse_url( $product['url'] );
 
+		// Bail if parsing the URL failed.
+		if ( $parsed_url === false ) {
```

Comment:
Just wanted to make sure this long form is purposeful (vs. `! $parsed_url`) and explicitly calling out `$parsed_url` might be a string or a bool.